src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/DWMColorization.cs

```diff
@@ -0,0 +1,682 @@
+using System;
+using System.Diagnostics;
+using System.Windows.Appearance;
+using System.Windows.Media;
+using Microsoft.Win32;
+using MS.Internal;
+using System.Runtime.InteropServices;
+
+namespace System.Windows;
+internal static class DWMColorization
+{
+    /// <summary>
+    /// Maximum <see cref="Byte"/> size with the current <see cref="Single"/> precision.
+    /// </summary>
+    private static readonly float _byteMax = (float)Byte.MaxValue;
```

Comment:
Can be `const` ?

Reply:
Yes, it can be const. I will go ahead and change the required.